addons/resource.language.en_gb/resources/strings.po

```diff
@@ -4961,7 +4969,7 @@ msgstr ""
 
 #: unknown
 msgctxt "#12362"
-msgid "Shutdown system if Master Lock retries exceeded"
+msgid "Shutdown system if \"Master lock\" retries exceeded"
```

Comment:
I'd change this to `Shutdown system if master lock retries is exceeded`. Compare to e.g. `The door will open if the "Master key" works`, to me that doesn't sound very good either.